docs/source/techniques/ISISPowder-OSIRIS-v1.rst

```diff
@@ -0,0 +1,307 @@
+.. _isis-powder-diffraction-osiris-ref:
+
+=====================================================
+ISIS Powder Diffraction Scripts - OSIRIS Reference
+=====================================================
+
+.. contents:: Table of Contents
+    :local:
+
+.. _creating_osiris_object_isis-powder-diffraction-ref:
+
+Creating OSIRIS Object
+------------------------
+This method assumes you are familiar with the concept of objects in Python.
+If not more details can be read here: :ref:`intro_to_objects-isis-powder-diffraction-ref`
+
+To create a OSIRIS object the following parameters are required:
+
+- :ref:`calibration_directory_osiris_isis-powder-diffraction-ref`
+- :ref:`output_directory_osiris_isis-powder-diffraction-ref`
+- :ref:`user_name_osiris_isis-powder-diffraction-ref`
+
+Optionally a configuration file may be specified if one exists
+using the following parameter:
+
+- :ref:`config_file_osiris_isis-powder-diffraction-ref`
+
+See :ref:`configuration_files_isis-powder-diffraction-ref`
+on YAML configuration files for more details
+
+Example
+^^^^^^^
+
+..  code-block:: python
+
+  from isis_powder.osiris import Osiris
+
+  calibration_dir = r"C:\path\to\calibration_dir"
+  output_dir = r"C:\path\to\output_dir"
+  config_dir = r"C:\path\to\config.yaml"
+
+  osiris_example = Osiris(user_name="Calib",
+                          calibration_directory=calibration_dir,
+                          output_directory=output_dir,
+                          config_file=config_dir)
+
```

Comment:
can we test the output?

Reply:
for this block there isn't really an output to test.